src/NuGet.Core/NuGet.Packaging/NuGetExtractionFileIO.cs

```diff
@@ -152,7 +152,7 @@ private static void ApplyUMaskToUnixPermissionsFromLibc()
             // read from running "sh -c umask", we're extremely unlikely to ever get here, so best-effort is good enough.
             var mask = Convert.ToInt32("700", 8);
             mask = PosixUMask(mask);
-            PosixUMask(mask);
+            _ = PosixUMask(mask);
```

Comment:
Unless I am missing something, this appears to be a redundant because `PosixUMask` is called in the above line.

Reply:
May be a follow-up issue for this one should be okay.

Reply:
I created follow up issue https://github.com/NuGet/Home/issues/11148

Reply:
When I wrote that code, I wrote 9 lines of comments, which explains why it needs to be called twice.